src/NativeScript/Metadata/Metadata.mm

```diff
@@ -207,8 +207,10 @@ int compareIdentifiers(const char* nullTerminated, const char* notNullTerminated
     } while (this->_topLevelIndex < this->_globalTable->buckets.count);
 }
 
+extern char startOfMetadataSection __asm("section$start$__DATA$__TNSMetadata");
```

Comment:
A CLI project statically links the runtime, unless you modify it to link against the runtime framework. Please make sure that it all works when linked against the framework, even double-check if necessary.

Maybe we can merge it as-is, but then we should make migrating away from asm a medium-priority issue.